utils/src/tree/AzExtTreeItem.ts

```diff
@@ -216,3 +215,7 @@ export abstract class AzExtTreeItem implements types.AzExtTreeItem {
         }
     }
 }
+
+export function isAzExtTreeItem(maybeTreeItem: unknown): maybeTreeItem is types.AzExtTreeItem {
+    return (maybeTreeItem as AzExtTreeItem)._isAzExtTreeItem === true;
```

Comment:
Might wanna test if it's an object first--`5._isAzExtTreeItem` would throw (I think?). In practice I think it's highly unlikely someone would use it in such a way, but better safe than sorry.

Reply:
```suggestion
    return typeof maybeTreeItem === 'object' && (maybeTreeItem as AzExtTreeItem)._isAzExtTreeItem === true;
```